src/main/scala/tile/RocketTile.scala

```diff
@@ -80,6 +81,111 @@ class RocketTile(
+          specifications = List[OMSpecification](PrivilegedArchitectureExtensions.specVersion(MachineLevelISA, "1.10")),
+          hasBasicCounters = coreParams.haveBasicCounters,
+          nAdditionalCounters = coreParams.nPerfCounters
+        ))
+      }
+      else { None }
+
+      val pmp = if (coreParams.pmpGranularity > 0 || coreParams.nPMPs > 0) {
+        Some(OMPMP(
+          specifications = List[OMSpecification](PrivilegedArchitectureExtensions.specVersion(MachineLevelISA, "1.10")),
+          nRegions = coreParams.nPMPs,
+          granularity = coreParams.pmpGranularity
+        ))
+      }
+      else { None }
+
+      val mulDiv = coreParams.mulDiv.map{ md => OMMulDiv.makeOMI(md, xLen)}
+
+      val baseInstructionSet = xLen match {
+        case 32 => if (XLen == 32) RV32E else RV32I // TODO coreParams.useRVE
+        case 64 => RV64I
+        case _ => throw new IllegalArgumentException(s"ERROR: Invalid Xlen: $xLen")
+      }
+
+      val isaExtSpec = ISAExtensions.specVersion _
+
+      val baseSpec = BaseExtensions.specVersion _
+
+      val baseISAVersion = baseInstructionSet match {
+        case RV32E => "1.9"
+        case RV32I => "2.0"
+        case RV64I => "2.0"
+        case _ => throw new IllegalArgumentException(s"ERROR: Invalid baseISAVersion: $baseInstructionSet")
+      }
+
+      val d = coreParams.fpu.filter(_.fLen > 32).map(x => isaExtSpec(D, "2.0"))
+
+      val omIsa = OMISA(
+        xLen = xLen,
+        baseSpecification = baseSpec(baseInstructionSet, baseISAVersion),
+        base = baseInstructionSet,
+        m = coreParams.mulDiv.map { x => isaExtSpec(M, "2.0") },
+        a = coreParams.useAtomics.option(isaExtSpec(A, "2.0")),
+        f = coreParams.fpu.map { x => isaExtSpec(F, "2.0") },
+        d = d,
+        c = coreParams.useCompressed.option(isaExtSpec(C," 2.0")),
+        u = coreParams.useUser.option(isaExtSpec(U,"1.10")),
+        s = None,
+        addressTranslationModes = Nil
+      )
+
+      val btb = rocketParams.btb.map(BTB.makeOMI(_))
+
+      val omICache = frontend.icache.device.getOMComponents(resourceBindingsMap).headOption.getOrElse(None).asInstanceOf[OMICache]
+
+      def dcache(p: DCacheParams): OMDCache = {
+        OMDCache(
+          memoryRegions = Nil,
+          interrupts = Nil,
+          nSets = p.nSets,
+          nWays = p.nWays,
+          blockSizeBytes = p.blockBytes,
+          dataMemorySizeBytes = p.nSets * p.nWays * p.blockBytes,
+          dataECC = p.dataECC.map(OMECC.getCode(_)),
+          tagECC = p.tagECC.map(OMECC.getCode(_)),
+          nTLBEntries = p.nTLBEntries
+        )
+      }
+
+      val omDCache = rocketParams.dcache.map(dcache(_))
+
+      Seq(OMRocketCore(
+        isa = omIsa,
+        mulDiv = mulDiv,
+        fpu = fpu,
+        performanceMonitor = perfMon,
+        pmp = pmp,
+        documentationName = "TODO",
+        hartIds = Seq(hartId),
+        hasVectoredInterrupts = true,
+        interruptLatency = 6,
```

Comment:
Where did this number come from and is it even correct?

Reply:
@hcook says we should remove the interruptLatency number from the docs.

Reply:
Yeah I have no idea what this number means

Reply:
Should interruptLatency be removed from the schema?

Reply:
No, this is important information to capture. The value should be 4 for rocket-chip.

Reply:
@hcook says that the number varies depending on the interrupt set. Lets go with 4 and then in rev2 lets revisit the way that the interrupt latency's are tracked in the OM.

Reply:
I just talked to @terpstra and @dbarbi1 and showed them this thread, and we think there's some confusion over what this value is supposed to be. This value is supposed to measure the latency between when an interrupt arrives at a core to when the core reacts to it, not the latency from the interrupt slaves to the core/interrupt controller.

Reply:
How about if we change the name of this variable to interruptCoreResponseLatency?
Is the latency 4 for rocket?
What is the latency for Caboose?